taipy/gui_core/_context.py

```diff
@@ -174,7 +174,10 @@ def submission_status_callback(self, submission_id: t.Optional[str]):
                             self.gui._call_user_callback(
                                 client_id,
                                 submission_name,
-                                [core_get(submission.entity_id), {"submission_status": new_status.name}],
+                                [
+                                    core_get(submission.entity_id),
```

Comment:
`core_get(submission.entity_id)` is a scenario. I guess we want to pass the submission instead: `core_get(submission.id)`.

@FredLL-Avaiga @toan-quach What do you think?

Reply:
the doc indicate : 1st param is a `submittable` (scenario or sequence)
We could add the `submission` to the detail but I think the main object is rightly the `submittable`.

Reply:
well, I'm not sure either so I don't have any comments on it

Reply:
This is my comment on this: I want two things generally. I want to know the Status of the job that has changed and the job itself. I also want to know if my whole submission has started, is canceled, or is completed. I want to see the progress of the jobs and how many of them are completed.

Reply:
@FlorianJacta Flo
in that case, I can add the submission entity as a metadata, what do you think?

Reply:
That could do the trick

Reply:
@jrobinAV @FredLL-Avaiga @FlorianJacta so do we all agree on:
- changing the `submittable` -> `submission` and  `submission.entity_id` -> `submission.id`? 
- adding `submission.entity_id` to the metadata under the key name `submittable_id`?

Reply:
I'm still not convinced but if everybody agrees...